core/modules/parser/FromFactory.cc

```diff
@@ -309,7 +309,7 @@ class FromFactory::RefGenerator {
     /// ( "inner" | outer_join_type ("outer")? )? "join" table_ref join_spec
     // Takes ownership of @param left
     query::JoinRef::Ptr _makeJoinWithSpec(RefAST sib) const {
-        if(!sib.get()) {
+        if (!sib.get()) {
             throw ParseException("Null JOIN_WITH_SPEC sibling", sib); }
```

Comment:
well, while you're at it, make either single line or move } to its own line here? :)